3p/integration.js

```diff
@@ -520,50 +510,75 @@ function installContextUsingExperimentalImpl(win) {
 /**
  * Installs window.context using standard (to be deprecated) implementation.
  * @param {!Window} win
+ * @param {!JsonObject} data
  */
-function installContextUsingStandardImpl(win) {
+function installContextUsingStandardImpl(win, data) {
+  const embedType = getEmbedType();
+  const contextState = getContextState();
+
+  win.context = dict({
```

Comment:
is `dict` better in anyway? would it cause type check problem for 3p code that is calling it using '.'?